keyboards/nomu30/config.h

```diff
@@ -20,30 +20,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xC0C0
-#define PRODUCT_ID      0x3000
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x3100
```

Comment:
Given you are adding a via keymap, you will want to use a non `FEED` vendor id (suggestion here is for RK and n30)
```suggestion
#define VENDOR_ID       0x524B
#define PRODUCT_ID      0x6E30
```

Reply:
Fixed bda7efc